presto/dataset.py

```diff
@@ -237,16 +290,32 @@ def __getitem__(self, idx):
 
 
 def filter_remove_noncrops(df: pd.DataFrame) -> pd.DataFrame:
-    crop_labels = [10, 11, 12, 13]
-    df = df.loc[df.LANDCOVER_LABEL.isin(crop_labels)]
+    labels_to_exclude = [
+        0,
+        991,
+        7900,
+        9900,
+        9998,  # unspecified cropland
+        1910,
+        1900,
+        1920,
+        1000,  # cereals, too generic
+        11,
+        9910,
+        6212,  # temporary crops, too generic
+        7920,
+        9520,
+        3400,
+        3900,  # generic and other classes
+        4390,
+        4000,
+        4300,  # generic and other classes
+    ]
+    df = df[(df["LANDCOVER_LABEL"] == 11) & (~df["CROPTYPE_LABEL"].isin(labels_to_exclude))]
```

Comment:
where does this list come from? And should we focus only on `LANDCOVER_LABEL == 11`?
These kind of methods have a pretty important impact downstream, so let's discuss them in depth to determine training strategy.